pkgs/applications/version-management/git-absorb/default.nix

```diff
@@ -2,20 +2,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "git-absorb";
-  version = "0.6.10";
+  version = "0.6.11";
 
   src = fetchFromGitHub {
     owner = "tummychow";
-    repo = pname;
+    repo = "git-absorb";
```

Comment:
This bit is not necessary.

Reply:
true, but using $pname in many places where they are not meant to be used and strings just match by coincidence is also not nice.